config/config.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    """ Default configurations """
+    DEBUG = False
+    TESTING = False
+    SQLALCHEMY_DATABASE_URI = "sqlite:///" + \
+        os.path.join(basedir, "bucketlist.db")
+
+
+class DevelopmentConfig(Config):
+    """ Development configurations """
+    DEBUG = True
+    TESTING = True
+    SQLALCHEMY_DATABASE_URI = "sqlite:///" + \
+        os.path.join(basedir, "bucketlist.db")
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SECRET_KEY = "p9Bv<3Eid9%$i01"
```

Comment:
Consider leaving your **SECRET_KEY** as an environment variable. Its unsafe to push it to a public repo, i.e,for future purposes.